### PR TITLE
Move certain GNOME3 options to programs

### LINKED ISSUE #66302: Rename several GNOME3 options
Currently these options, after reflection, are named confusingly because they're aren't useful as a standalone "service" and are better recognized as programs/applications.

* `services.gnome3.file-roller.enable` -> `programs.file-roller.enable`
* `services.gpaste.enable` -> `programs.gpaste.enable`
* `services.evince.enable` -> `programs.evince.enable`
* `services.gnome3.gnome-disks.enable` -> `programs.gnome-disks.enable`
* `services.gnome3.gnome-documents` -> `programs.gnome-documents`

Not sure if `services.gnome3.sushi` should be a `programs`.

---

Do you think it's worth it to rename these like this?
It's already confusing that you have to use a NixOS module
to use certain packages. Could we devise something better here?

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
Fixes #66302

I noticed while moving `gpaste` that it used `sessionPath`.
Do we still need that @jtojnar?

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by jtojnar** [comment]
I do not think it should be necessary. I even removed it in https://github.com/NixOS/nixpkgs/commit/f63d94eba340c05cd241c478deb518204514fce8. No sure how it got back.

**Comment by worldofpeace** [comment]
Weird, GitHub says that change never touched that file
* https://github.com/NixOS/nixpkgs/commits/master/nixos/modules/services/desktops/gnome3/gpaste.nix

Will fix that here.

File: nixos/modules/programs/gpaste.nix [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 { config, lib, pkgs, ... }:
 
 with lib;
 
 {
+
+  # Added 2019-08-09
+  imports = [
+    (mkRenamedOptionModule
+      [ "services" "gpaste" "enable" ]
```
Comment (jtojnar) [inline nixos/modules/programs/gpaste.nix]:
```suggestion
      [ "services" "gnome3" "gpaste" "enable" ]
```

**Comment by worldofpeace** [comment]
Did that @jtojnar https://github.com/NixOS/nixpkgs/pull/66398/commits/7e3c8fa8c579e42eb7a15c47208dd95836fe3f71

**Review by jtojnar — APPROVED** [review]
Good, seems to work now.

```
building the system configuration...
trace: warning: The option `services.gnome3.gpaste.enable' defined in `/home/jtojnar/Projects/nix-playground/gdm-fail.nix' has been renamed to `programs.gpaste.enable'.
```

**Comment by worldofpeace** [comment]
Thanks @jtojnar

**Pull request merged**